Notes panel

manognyaa opened the pull request:
## Related Issue 
I have created Notes panel so that everyone can easily note down the points.


- [x] LGM-SOC'21 Participant 
- [ ] Contributor

Closes: #237 

### Describe the changes you've made

Give a clear description what modifications you have made

## Type of change

What sort of change have you made:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Once, you download the code in your local computer, run it. It will work successfully.

## Checklist:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
![screen-capture (20)](https://user-images.githubusercontent.com/77045147/122015852-1c876080-cdde-11eb-90a9-e05ed88cf710.gif)
![notes 1](https://user-images.githubusercontent.com/77045147/122015874-214c1480-cdde-11eb-83eb-fd4f861c60f9.png)
![notes 2](https://user-images.githubusercontent.com/77045147/122015903-2741f580-cdde-11eb-998d-eae244d92152.png)

| Updated Screenshot |
please guide me if any changes required @prathimacode-hub @Debashish-hub .

prathimacode-hub reviewed (changes requested):
Small changes. 

- The project folder should be titled as "Notes Panel"
- The README should be as "README.md"
- Add the libraries imported in "requirements.txt" file.

Kindly do the needful changes. @manognyaa

manognyaa commented:
> Small changes.
> 
> * The project folder should be titled as "Notes Panel"
> * The README should be as "README.md"
> * Add the libraries imported in "requirements.txt" file.
> 
> Kindly do the needful changes. @manognyaa

hello mam,
i have updated the README.md file, titled folder as Notes Panel and added the requirements.txt file

please guide me, if any changes are required @prathimacode-hub

prathimacode-hub reviewed (approved):
Your PR is approved. You showed lot of improvement from your previous PR. I appreciate your work. Amazing. 🏷🙌@manognyaa
